_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {hasError: false};
```

Comment:
Here ad in FullName, we need to whitelist the valid characters even for non-required fields.  "#$@#**" is not a valid name.